Jenkinsfile.flake8

```diff
@@ -14,8 +14,8 @@ def label = "salt-flake8-${UUID.randomUUID().toString()}"
 
 podTemplate(label: label, containers: [
         containerTemplate(
-            name: 'opensuse',
-            image: 'opensuse:42.3',
+            name: 'tox',
+            image: 'registry.suse.de/devel/casp/ci/opensuse_leap_42.3_containers/jenkins-tox-container:latest',
```

Comment:
Leap 42.3 is planned for EOL in June, which was much farther away last May when this was first written. :)  Any idea how big a deal is it to move this to something based on Leap 15?